search/lib/application.js

```diff
@@ -2,10 +2,17 @@ const express = require('express');
 const awsServerless = require('aws-serverless-express');
 const awsServerlessMiddleware = require('aws-serverless-express/middleware');
 
+const settings = require('./settings');
+const { createLogger } = require('./util');
+const logger = createLogger(settings.logger);
+
 const api = require('./api');
 const { errorHandler } = require('./error-handler');
 
 async function initialize () {
+  logger.debug('Initialize Application');
+  logger.silly('This is ridiculous.');
```

Comment:
I think we can remove this line as it was just for an example. I would probably remove the line above as well unless you're adding more context.

Reply:
yup, I will remove the silly message.